en/02_Developer_Guides/03_Forms/01_Validation.md

```diff
@@ -394,6 +394,39 @@ class MyPage extends Page
 > You can also update the `CompositeValidator` by creating an `Extension` and implementing the
 > `updateCMSCompositeValidator()` method.
 
+### `RequiredFields` and whitespace
+
+By default, `RequiredFields` will consider a field with only whitespace as a valid value. You an change this behavior with the [`allow_whitespace_only`](api:SilverStripe\Forms\RequiredFields->allow_whitespace_only) global configuration, or on a per-instance basis using [`setAllowWhitespaceOnly()`](api:SilverStripe\Forms\RequiredFields::setAllowWhitespaceOnly()).
```

Comment:
Don't forget to update these references for the CMS 6 docs

Reply:
Added a reminder to https://github.com/silverstripe/developer-docs/pull/645#issue-2713937820